templates/plugin_impl_cpp/stream.j2

```diff
@@ -1,7 +1,10 @@
 {% if is_async %}
-void {{ plugin_name.upper_camel_case }}::{% if not is_finite %}subscribe_{% endif %}{{ name.lower_snake_case }}{% if is_finite %}_async{% endif %}({% for param in params %}{{ param.type_info.name }} {{ param.name.lower_snake_case }}, {% endfor %}{{ name.upper_camel_case }}Callback callback)
+void {{ plugin_name.upper_camel_case }}Impl::{% if not is_finite %}subscribe_{% endif %}{{ name.lower_snake_case }}{% if is_finite %}_async{% endif %}({% for param in params %}{{ plugin_name.upper_camel_case }}::{{ param.type_info.name }} {{ param.name.lower_snake_case }}, {% endfor %}{{ plugin_name.upper_camel_case }}::{{ name.upper_camel_case }}Callback callback)
 {
-    _impl->{{ name.lower_snake_case }}_async({% for param in params %}{{ param.name.lower_snake_case }}, {% endfor %}callback);
+    {% for param in params %}
+    UNUSED({{ param.name.lower_snake_case }});
```

Comment:
Not sure I get this. Now the function is not doing anything anymore, is it?

Reply:
Right, it's just the skeleton and you have to fill in the impl implementation (pun intended).